src/Go/boardAnalysis/example.js

```diff
@@ -1,3 +1,9 @@
+/* eslint-disable */
```

Comment:
I'm fine with this, but we might want to delete this or move this file elsewhere.